lib/LambdaHelper.js

```diff
@@ -152,4 +232,57 @@ module.exports = function (aws, queueInitializedPromise, executionStore){
 
         return proxyStore[functionIdentifier];
     };
+
+    this.getCloudedFunctionFromModule = function(modulePath, handlerName, deps, configs) {
```

Comment:
True about the deps but for the functionConfigs the name is optional so the ```extend``` will cover the ```null || undefined``` case